src/redux/books/books.js

```diff
@@ -0,0 +1,27 @@
+const ADD_BOOK = 'bookStore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookStore/books/REMOVE_BOOK';
+
+const initialState = [{ title: 'title demo', category: 'category demo', id: 'demo id' }];
```

Comment:
- [ ] As it is a requirement for this project, please your `initial state` should be an empty array. Kindly fix that.
![initial](https://user-images.githubusercontent.com/65852621/146627681-d9e2fe09-37b0-4897-a8f8-850122025e0f.png)